src/RA_Dlg_MemBookmark.cpp

```diff
@@ -758,17 +751,19 @@ void Dlg_MemBookmark::ExportJSON()
                                     doc.AddMember("Bookmarks", bookmarks, allocator);
```

Comment:
I did do a format pass on this using the provided vssettings file, GitHub skips a few lines, click "View whole File" if you need to see it for yourself.